### ISSUE TITLE
Page break

### ISSUE THREAD

**Issue by naivenaive** [VolodymyrBaydalka/docxjs#39]
Thanks for your excellent work. I found a feature not documented in README. If there is not any page break inserted in the docx file, all content will be rendered in one page no matter how long it is.  Is there any option to enable page break automatically just like Microsoft office?

**Comment by lwq-js** [comment]
idem

**Comment by Timson020** [comment]
same problem

can you help me. ?

**Comment by VolodymyrBaydalka** [comment]
> Is there any option to enable page break automatically just like Microsoft office?

No, there is no such option. Proper insertion page breaks automatically will require re-calcultion of sizes of each and every element of document on each and every change (element insertion, style changes, etc) and it may affect perfomance a lot.

**Comment by Kn3cht** [comment]
Any updates on this?

**Comment by w-z-y** [comment]
Any updates on this?

**Comment by thuongvovan** [comment]
I use this fork. This work fix this issuse
https://github.com/millet0328/docx-preview-sync
